_posts/2018-08-30-disability-at-rustfest.md

```diff
@@ -0,0 +1,20 @@
+---
+layout: post
+title: "Disability @ RustFest"
+#socialImageSrc: "/assets/posts/rome.jpg"
+authors:
+  - skade
+  - katsel
+---
+
+RustFest Rome has a special program for disabled Rustaceans. First of all, check our [accessibility statement](https://rome.rustfest.eu/accessibility/) for general information about the accessibility of our venue and our already present accessibility features, such as live transcription. If there is anything missing that you need, please get in touch with us as early as possible. Also, you may want to read about [Diversity @ RustFest](https://rome.rustfest.eu/diversity).
```

Comment:
> First of all, check our…

First of all, please check our…

> Also, you may want to read about [Diversity @ RustFest](https://rome.rustfest.eu/diversity)

Also, you may want to read about [diversity at RustFest](https://rome.rustfest.eu/diversity)